### PR TITLE
For issue #162 Add different regex pattern to search for meta tags

### PR DESCRIPTION
**Pull request by AngelikiBoura**
Changes:

- Add a second regex pattern with different ordering in meta tags
- Test that redirection occurs when dtags are differently ordered

### CODE REVIEW

File: tests/test_html.py [hunk 1]
```diff
@@ -669,3 +669,14 @@ def test_inside_script(self):
             get_meta_refresh(body, baseurl, ignore_tags=()),
             (0.0, "http://example.org/foobar_required"),
         )
+
+    def test_redirections_in_different_ordering__in_meta_tag(self):
+        baseurl = 'http://localhost:8000'
```
Comment (andreastziortz) [inline tests/test_html.py:674]: Nice test! I guess we could add more examples if necessary

File: w3lib/html.py [hunk 2]
```diff
@@ -21,6 +21,26 @@
     r'<meta\s[^>]*http-equiv[^>]*refresh[^>]*content\s*=\s*(?P<quote>["\'])(?P<int>(\d*\.)?\d+)\s*;\s*url=\s*(?P<url>.*?)(?P=quote)',
     re.DOTALL | re.IGNORECASE,
 )
+_meta_refresh_re2 = re.compile(
```
Comment (andreastziortz) [inline w3lib/html.py:24]: I like that, but I think we should somehow drop all logic into the same regex pattern

File: w3lib/html.py [hunk 3]
```diff
@@ -338,8 +358,9 @@ def get_meta_refresh(
         raise
     utext = remove_tags_with_content(utext, ignore_tags)
     utext = remove_comments(replace_entities(utext))
-    m = _meta_refresh_re.search(utext)
+    m = _meta_refresh_re.search(utext) or _meta_refresh_re2.search(utext)
```
Comment (andreastziortz) [inline w3lib/html.py:346]: I guess here will be one search pattern that we should use.

File: w3lib/html.py [hunk 4]
```diff
@@ -338,8 +358,9 @@ def get_meta_refresh(
...
     if m:
+        m.group("int")
```
Comment (andreastziortz) [inline w3lib/html.py]: I guess we could remove that line

**Comment by andreastziortz** [comment]
I am keen on the idea but I think we could make some improvements. Overall, I agree with your idea

**Review by andreastziortz — CHANGES_REQUESTED** [review]
I am keen on the idea, but I think we could make some improvements.

**Comment by AngelikiBoura** [comment]
@andreastziortz I liked the idea of the common regex pattern as well but i haven't found yet a way to make it work.
Since the code is functional now,  shall we make a PR to the main repository and listen to their feedback?

**Comment by andreastziortz** [comment]
Fine with me!

**Pull request merged**